cmd/minikube/cmd/root.go

```diff
@@ -77,7 +80,7 @@ func init() {
 // initConfig reads in config file and ENV variables if set.
 func initConfig() {
 	viper.SetConfigName("config")
-	viper.AddConfigPath(constants.ConfigFilePath)
+	viper.AddConfigPath(constants.MakeMiniPath("config"))
```

Comment:
I got rid of the constants.ConfigFilePath as no other MiniPath directory had a specified value in constants, they all used there 'name' as a parameter to MakeMiniPath.  I was trying to make 'config' follow this standard.